### ISSUE TITLE
As an Industry Reporter I want to see the number of Facilities I have for each Operation so that I ensure the correct # of Facilities are associated with that Operation

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#233]
#### Description:

This has to do with adding the 'No. of facilities' column in the reporting dashboard. We need to display how many facilities are assigned to an operation at the level of registration.
Note: If an operator has not properly registered their facilities, this column will be inaccurate or empty. 

[Wireframe here](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7789%3A2031&t=JjzBvwOdyb7G91xp-1)

Blocked by: https://github.com/bcgov/cas-reporting/issues/243

#### Acceptance Criteria:

Given I am an Industry Reporter
When I am on the Reporting Dashboard 
Then I will see the number of facilities that are associated/registered to each operation listed

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

- We may need to confirm this through User Testing - is this column useful?

**Comment by patriciarussellCAS** [comment]
Tagging @pbastia for refinement

**Comment by patriciarussellCAS** [comment]
Flagging that this may come up in Ext. User testing - we would like to confirm that seeing the # of facilities listed beside each Operation is helpful/what they want to see here. If yes, then we can prioritize this ticket. If not, we can icebox it.

**Comment by patriciarussellCAS** [comment]
@suhafa - wondering your thoughts on this column in the reporting dashboard. I'm thinking we could leave this out of the dashboard, and just have ticket #197 for LFO's - where they review their list of facilities. I got the impression that seeing the full list of facilities was more important to users than seeing the # listed. 

We could keep it in the backlog but de-prioritize for now? 

cc: @pbastia & @dleard
